src/lib/features/feature-toggle/tests/feature-toggles.e2e.test.ts

```diff
@@ -3609,11 +3609,16 @@ test('Updating feature strategy sort-order should trigger a an event', async ()
     );
 
     const strategies: FeatureStrategySchema[] = body;
-    let order = 1;
     const sortOrders: SetStrategySortOrderSchema = [];
 
-    strategies.forEach((strategy) => {
-        sortOrders.push({ id: strategy.id!, sortOrder: order++ });
```

Comment:
Increasing all sortOrders by one was actually not changing the order of the strategies, so the event was not emitted after this change